tests/test_cache_manager.py

```
@@ -0,0 +1,328 @@
+"""
+Testes para o CacheManager - Sistema de cache otimizado.
+"""
+
+import pytest
+import tempfile
+import time
+import threading
+import os
+from pathlib import Path
+from unittest.mock import patch
```

Ruff (0.12.2)

`unittest.mock.patch` imported but unused

Remove unused import: `unittest.mock.patch`

(F401)